cmd/gb-list/main.go

```diff
@@ -47,9 +56,27 @@ func main() {
 	if err != nil {
 		gb.Fatalf("unable to resolve: %v", err)
 	}
-	for _, pkg := range pkgs {
-		if err := tmpl.Execute(os.Stdout, pkg); err != nil {
-			gb.Fatalf("unable to execute template: %v", err)
+
+	if jsonOutput {
+		views := make([]*PackageView, 0, len(pkgs))
+		for _, pkg := range pkgs {
+			views = append(views, NewPackageView(pkg))
+		}
+		encoded, err := json.MarshalIndent(views, " ", "  ")
```

Comment:
I will try again. I was not able to find a way to write prettified JSON with pure json encoder.